tools/extra/qpafilter/test_qpafilter.py

```diff
@@ -0,0 +1,74 @@
+import pytest
```

Comment:
Missing header at the top, imports should be done in order.

Reply:
fixed!